ci-operator/config/red-hat-storage/df-console/red-hat-storage-df-console-main.yaml

```diff
@@ -0,0 +1,69 @@
+base_images:
+  base:
+    name: release
+    namespace: openshift
+    tag: tectonic-console-builder-v23
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: tectonic-console-builder-v23
+images:
+  - dockerfile_path: Dockerfile.ci
+    inputs:
+      base:
+        as:
+          - node:15 # Do we support later versions for df-console?
+    to: df-console
+releases:
+  initial:
+    integration:
+      name: "4.10"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.10"
+      namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+  - as: lint
+    steps:
+      test:
+        - as: lint
+          commands: yarn install && yarn run lint
+          from: src
+          resources:
+            requests:
+              cpu: 100m
+              memory: 200Mi
+  - as: df-console-e2e-aws
+    steps:
+      cluster_profile: aws
+      env:
+        COMPUTE_NODE_TYPE: m4.2xlarge
+        ZONES_COUNT: "1"
+      test:
+        - as: tests
+          cli: latest
+          commands: ./test-prow-e2e.sh;
```

Comment:
need to pass image name...

Reply:
We'll hardcode the image name since it's the same.